lib/userFactory.js

```diff
@@ -69,6 +69,9 @@ class UserFactory extends BaseFactory {
      * @return {UserFactory}
      */
     static getInstance(config) {
+        if (!instance && (!config || !config.scmPlugin)) {
+            throw new Error('No scm plugin provided to UserFactory');
```

Comment:
@FenrirUnbound Yes, they just moved the scmPlugin check from base to userFactory. So the functionality is unchanged.